src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -162,11 +166,10 @@ class JesBackend extends Backend with LazyLogging {
     val jesParameters = standardParameters(callGcsPath) ++ jesInputs ++ unsafeJesOutputs
 
     // Not possible to currently get stdout/stderr so redirect everything and hope the WDL isn't doing that too
-    val redirectedCommand = s"$commandLine > $LocalStdoutValue  2> $LocalStderrValue"
+    val redirectedCommand = s"$instantiatedCommandLine > $LocalStdoutValue 2> $LocalStderrValue"
```

Comment:
I believe running this as a command script the way the local backend does would get around this problem (write a script, stage it up, include it among the files to be localized, sh -c the script).  This approach was also suggested by Google as a solution to the command line length problem.